config/common/story_cycles.cwt

```diff
@@ -10,6 +10,15 @@ story_cycle = {
     on_owner_death = single_alias_right[effect_clause]
     ## cardinality = 0..inf
     effect_group = {
+        ## cardinality = 0..1
+        enum[duration] = int
```

Comment:
I'm not exactly clear yet on the difference between `int` and `int_value_field`.  `enum[duration]` actually uses both throughout this project.  Let me know if this should be changed.

Reply:
I think the difference is that `int_value_field` allows references to e.g. script values (`add_tyranny = some_scripted_value_name`), while `int` accepts only a plain number (`add_tyranny = 420`).

If you haven't noticed any story cycle that uses a named reference for duration value, I think it's OK to keep `int`.